### PR TITLE
Reusable tag ui

### LINKED ISSUE #49: Reusable Tag UI
Create a reusable component we'll call "Tag". It should:

* Have a title
* Have a selected state and a deselected state
* Be tappable
* Tapping should toggle between deselected and selected states

**Design**
Check on the screenshot below
[Link to Design](https://www.figma.com/file/4cV84H1jyMo0rJEAl3HNnJ/FightPandemics-landing-page?node-id=6638%3A29348)

**VARIATIONS**:

Screen Shot 2020-05-13 at 9 41 14 PM

### PR DESCRIPTION
**Pull request by eddyteddy3**
**What this PR does**:

-  Added Shared Folder for the Reusable UI Components
-  Added Reusable Tag UI Class.

### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #

**Special notes for reviewers**:
I have made Target for the TapGesture within the class, you may alter it according to the usage. 

**Additional comments**:

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -11,6 +11,6 @@ SPEC REPOS:
 SPEC CHECKSUMS:
   SwiftLint: 22ccbbe3b8008684be5955693bab135e0ed6a447
 
-PODFILE CHECKSUM: f12283feb35e1e67ec7946a36c30e748cd6ac057
+PODFILE CHECKSUM: 9e9fa74647c6b2fb89bbf8f6a7f0102a43ae068b
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```
Comment (haidalinda) [inline Podfile.lock]: would it be possible to update your CocoaPods. version to `1.9.1` @eddyteddy3  - otherwise we'll constantly be getting this mismatch

**Comment by haidalinda** [comment]
@eddyteddy3 please pay attention to this checklist before you make your PRs: 

```
[] Compiler warnings resolved
[] Linter warnings resolved
[] User-facing strings in Localizable.strings file
[] Unused code, print statements, and comments removed
```

and fill it out! even if youre PR easily meets this criteria, don't ignore 🙏

**Comment by haidalinda** [comment]
also @eddyteddy3 - in your PR, be sure to fill out the area `Which issue(s) this PR fixes:`; that helps keep our Project board up-to-date

File: Podfile.lock (hunk 1 above)
Comment (eddyteddy3) [reply Podfile.lock]: Yes, I have updated my CocoaPods Version :)

**Comment by eddyteddy3** [comment]
I have updated the PR.

**Review by stavares843 — APPROVED** [review]

**Review by haidalinda — APPROVED** [review]

**Pull request merged**